addons/tmf_loadouts/loadouts/uk_rm_1982.hpp

```diff
@@ -0,0 +1,517 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* Community Factions Project
+//		* FIR AWS(AirWeaponSystem)
+//		* MNP Uniforms
+//		* HLC FAL Pack
+//		* HLC MP5 Pack
+//		* USM
+// 	* REDD Milan
```

Comment:
Missing a tab

Reply:
Weird; when in notepad++ it is showing as having two tabs. Fix'd?